corehq/apps/reports/tasks.py

```diff
@@ -121,11 +123,15 @@ def get_report_queue(report):
 @task(ignore_result=True)
 def send_report(notification_id):
     notification = ReportNotification.get(notification_id)
+    owner = WebUser.get(notification.owner_id)
```

Comment:
seems ok since this runs in the background and the reports themselves are likely way slower.